galaxy_ng/tests/unit/api/test_api_ui_container_registry_remote.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from rest_framework import status
+from django.test import override_settings
+
+from galaxy_ng.app.models import auth as auth_models
+
+from galaxy_ng.app.constants import DeploymentMode
+
+from .base import BaseTestCase, get_current_ui_url
+
+log = logging.getLogger(__name__)
+
+
+@override_settings(GALAXY_DEPLOYMENT_MODE=DeploymentMode.STANDALONE.value)
+class TestContainerRegistryRemoteViewset(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.admin_user = auth_models.User.objects.create(username='admin',
+                                                          is_superuser=True)
+        self.pe_group = self._create_partner_engineer_group()
+        self.admin_user.groups.add(self.pe_group)
+        self.admin_user.save()
+
+    def test_get_registry_list_empty(self):
+        url = get_current_ui_url('execution-environments-registry-list')
+        self.client.force_authenticate(user=self.admin_user)
+        response = self.client.get(url)
+        self.assertEqual(response.data['data'], [])
+
+    def create_registry_remote(self, name):
+        url = get_current_ui_url('execution-environments-registry-list')
+        new_registry = {
+            'name': name,
+            'url' : 'http://example.com',
+        }
+        self.client.force_authenticate(user=self.admin_user)
+        response = self.client.post(url, new_registry, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        return response.data['pk']
+
+    def test_get_detail_with_data(self):
+        name = 'my_registry'
+        pk = self.create_registry_remote(name)
+        url = get_current_ui_url('execution-environments-registry-detail', kwargs={'pk': pk})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['name'], name)
```

Comment:
I recommend testing at at least one other field. You could test something with the `last_sync_task` field, that would actually fail when you rename the `ui/serializers/execution_environment.py` method from `last_sync_task_queryset()` to `last_sync_task()` which is really helpful - this test could check the format is as expected.

It would probably look something like this: `self.assertIn("task_id", response.data['last_sync_task'])`

Reply:
The registry that we made with the helper function has a 'last_sync_task' of None, should I just assign it a number? or generate it some other way?

Reply:
@ShaiahWren not sure if you saw my other comment about `get_last_sync_task_queryset()`... in the django api page see what the `last_sync_task` field looks like before / after that change

Reply:
![Screen Shot 2021-08-23 at 2 28 54 PM](https://user-images.githubusercontent.com/64337863/130688463-52ae6b7b-57d2-457a-9c67-c5daad10e76b.png)

Reply:
Seems we cannot test format of sync task here in tests since a sync task will not exist, this test LGTM